lib/adal-angular.js

```diff
@@ -320,7 +320,7 @@ if (typeof module !== 'undefined' && module.exports) {
                             if (authService.loginInProgress()) {
                                 authService.info('login already start.');
                                 return $q.reject();
-                            } else if (authService.config && isEndpoint) {
+                            } else if (authService.config && (!authService.config.endpoints || isEndpoint)) {
```

Comment:
I'm a little bit concerned doing this way. What we really want to do is checking if it's in endpoint list or from app backend. But we already return if it's not from app backend or in endpoint list when checking the resource. Let me give a little bit more thought on this and get back to you.